config/default/llmisvc_validatingwebhook_cainjection_patch.yaml

```diff
@@ -3,6 +3,4 @@ kind: ValidatingWebhookConfiguration
 metadata:
   name: llminferenceservice.serving.kserve.io
   annotations:
-    cert-manager.io/inject-ca-from: $(kserveNamespace)/serving-cert
-webhooks:
-  - name: llminferenceservice.kserve-webhook-server.validator
\ No newline at end of file
+    cert-manager.io/inject-ca-from: $(kserveNamespace)/serving-cert
```

Comment:
follow-up: how about removing `name` that narrows the patch from other patches for future-proofing?